src/components/SkipButton/SkipButton.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import SkipButton from './SkipButton';
+
+describe(SkipButton, () => {
```

Comment:
How the title here is being used is the component itself and not a string? SkipButton

Reply:
Jest is smart enough to pick up the name from the function or class unless they are memoized